src/Variable.ts

```diff
@@ -13,31 +17,25 @@ export default class Variable {
   public sliderMax: number = 100;
   public isDiscrete: boolean = true;
 
-  public id: string;
+  constructor(options: {
+    name: string;
+    id?: string;
+
+    value: ScalarValue;
+    cloud?: boolean;
 
-  constructor(
-    options: {
-      name: string;
-      value: number | string;
-      cloud?: boolean;
-
-      visible?: boolean;
-      mode?: "default" | "slider" | "large";
-      x?: number;
-      y?: number;
-      sliderMin?: number;
-      sliderMax?: number;
-      isDiscrete?: boolean;
-    },
-    id?: string
-  ) {
+    visible?: boolean;
+    mode?: "default" | "slider" | "large";
+    x?: number;
+    y?: number;
+    sliderMin?: number;
+    sliderMax?: number;
+    isDiscrete?: boolean;
```

Comment:
Will variables (and lists, for that matter) ever be initialized without these values provided?

Reply:
Probably not, but otherwise the default values listed above mean nothing.
/cc @PullJosh for thoughts here!